src/tbbmalloc/large_objects.h

```diff
@@ -81,18 +81,25 @@ struct HugeBinStructureProps {
 
     static size_t alignToBin(size_t size) {
         MALLOC_ASSERT(size >= StepFactor, "Size must not be less than the StepFactor");
-        size_t minorStepExp = BitScanRev(size) - StepFactorExp;
+
+        int sizeExp = (int)BitScanRev(size);
+        MALLOC_ASSERT(sizeExp >= 0, "Size must be greater than 0");
+        MALLOC_ASSERT(sizeExp >= StepFactorExp, "sizeExp must not be less than the StepFactorExp");
```

Comment:
Can it be like "sizeExp >= StepFactorExp, because size >= StepFactor"

Reply:
Done